src/app/builds/build-details/build-details.component.ts

```diff
@@ -23,18 +23,22 @@ export class BuildDetailsComponent implements OnInit, OnDestroy, AfterViewChecke
 
   constructor(
     private route: ActivatedRoute,
+    private projectService: ProjectService,
     private buildService: BuildService,
     private configService: ConfigService,
     private titleService: Title) { }
 
   ngOnInit(): void {
-    this.buildId = this.route.snapshot.paramMap.get('buildId');
-    this.buildService.buildBuildidGet(Number(this.buildId)).subscribe(build => {
+    this.buildId = Number(this.route.snapshot.paramMap.get('buildId'));
+    this.buildService.buildBuildidGet(this.buildId).subscribe(build => {
       this.buildStatus = build.statusId;
       this.connect();
     });
-
-    this.updateTitle();
+    const projectId = Number(this.route.snapshot.paramMap.get('projectId'));
+    this.projectService.projectProjectidGet(projectId).subscribe(project => {
+      this.project = project;
+      this.updateTitle();
+    });
   }
```

Comment:
What would be an easy way to get the project name here that does not involve requesting it from the wharf-api?